qemu/tests/hpt_miscellaneous.py

```diff
@@ -118,4 +118,12 @@ def run(test, params, env):
         session = vm.reboot(session)
         vm.verify_alive()
         verify_hpt(test, params, session, hpt_default)
-    vm.verify_kernel_crash()
+    if "migration" in params.get("sub_type"):
+        set_hpt(session, params, test, hpt_size)
```

Comment:
No need to set it again, it has been set in line 115.

Reply:
sure, will adjust it and thanks.